### PR TITLE
Mention removal of AndroidX Core

### PR DESCRIPTION
**Pull request by marandaneto**
similar to https://docs.sentry.io/platforms/android/configuration/releases/#androidx-support

### CODE REVIEW

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -158,6 +158,20 @@ Unresponsive UI and animation hitches annoy users and degrade the user experienc
...
+
+We check at runtime for availability, so if you're not using AndroidX.Core, you can remove it from Sentry's transitive dependencies.
+
+```Groovy
+implementation ('io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.2.0') }}') {
+    exclude group: 'androidx.core', module: 'core'
+}
+```
+
+Be aware that by removing this transitive dependency, the Slow and Frozen Frames won't work.
```
Comment (romtsn) [inline src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:173]:
```suggestion
Be aware that by removing this transitive dependency, the Slow and Frozen Frames won't be reported.
```

**Review by romtsn — APPROVED** [review]

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -158,6 +158,20 @@ Unresponsive UI and animation hitches annoy users and degrade the user experienc
 
 Slow and frozen frames are Mobile Vitals, which you can learn about in the [full documentation](/product/performance/mobile-vitals).
 
+### AndroidX Support
+
+Sentry uses the AndroidX.Core library for detecting Slow and Frozen Frames. This is necessary for having an accurate result across all the Android OS versions.
```
Comment (imatwawana) [inline src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:163]:
```suggestion
Sentry uses the AndroidX.Core library for detecting [slow and frozen frames](#slow-and-frozen-frames). This is necessary to produce accurate results across all Android OS versions.
```

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 3]
```diff
@@ -158,6 +158,20 @@ Unresponsive UI and animation hitches annoy users and degrade the user experienc
...
+
+We check at runtime for availability, so if you're not using AndroidX.Core, you can remove it from Sentry's transitive dependencies.
```
Comment (imatwawana) [inline src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:165]:
```suggestion
We check for availability at runtime, so if you're not using AndroidX.Core, you can remove it from Sentry's transitive dependencies.
```

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 4]
```diff
@@ -158,6 +158,20 @@ Unresponsive UI and animation hitches annoy users and degrade the user experienc
...
+
+```Groovy
+implementation ('io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.2.0') }}') {
+    exclude group: 'androidx.core', module: 'core'
+}
+```
+
+Be aware that by removing this transitive dependency, the Slow and Frozen Frames won't be reported.
```
Comment (imatwawana) [inline src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:173]:
```suggestion
Note that by removing this transitive dependency, slow and frozen frames won't be reported.
```

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 5]
```diff
@@ -158,6 +158,20 @@ Unresponsive UI and animation hitches annoy users and degrade the user experienc
...
+
 ### OkHttp library Instrumentation
```
Comment (imatwawana) [inline src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:175]:
```suggestion
### OkHttp Library Instrumentation
```

**Review by imatwawana — APPROVED** [review]
Minor wording edits

**Comment by marandaneto** [comment]
thanks a lot @imatwawana

**Pull request merged**